src/core_codemods/use_set_literal.py

```diff
@@ -0,0 +1,28 @@
+import libcst as cst
+
+from codemodder.codemods.api import BaseCodemod, ReviewGuidance
+from codemodder.codemods.utils_mixin import NameResolutionMixin
+
+
+class UseSetLiteral(BaseCodemod, NameResolutionMixin):
+    NAME = "use-set-literal"
+    SUMMARY = "Use Set Literals Instead of Sets from Lists"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_WITHOUT_REVIEW
+    DESCRIPTION = "Replace sets from lists with set literals"
+    REFERENCES: list = []
+
+    def leave_Call(self, original_node: cst.Call, updated_node: cst.Call):
+        match original_node.func:
+            case cst.Name("set"):
+                if self.is_builtin_function(original_node):
+                    match original_node.args:
+                        case [cst.Arg(value=cst.List(elements=elements))]:
+                            # Can't use set literal for empty set
+                            if len(elements) == 0:
+                                self.add_change(original_node, self.CHANGE_DESCRIPTION)
```

Comment:
should also be able to call `self.report_change(original_node)` which I think is nicer but they're kinda dupes of each other